serverless/index-serverless-project-settings.asciidoc

```diff
@@ -25,3 +27,16 @@ include::./pages/transforms.asciidoc[leveloffset=+3]
 include::./pages/integrations.asciidoc[leveloffset=+2]
 
 include::./pages/fleet-and-elastic-agent.asciidoc[leveloffset=+2]
+
+include::./pages/manage-org.asciidoc[leveloffset=+2]
+
+include::./pages/manage-your-project.asciidoc[leveloffset=+2]
```

Comment:
We can rename this to Manage performance and data retention so it reflects better the content within this updated hierarchy